boot/zephyr/Kconfig

```diff
@@ -466,6 +474,14 @@ config BOOT_SERIAL_DETECT_PIN_VAL
 	  Logic value of the detect pin which triggers serial recovery
 	  mode.
 
+config BOOT_SERIAL_DETECT_DELAY
+	int "Prevents immediate boot into MCUBoot firmware."
```

Comment:
maybe "Serial detect pin denounce time [ms]"

Reply:
Ahh yea that would make more sense huh. I pushed a tweak to this. Let me know if it looks ok.